### PR TITLE
implement raw_solr_query - closes #7

### LINKED ISSUE #7: find_all querying
I am querying a geospatial index with the following code snippet but not getting any results back.

``` js
collection.find_all({
    d: '100000',
    pt: '42,-12',
    sfield: 'location'
}, console.log);
```

with the following data in my collection

``` js
{"location":"42,-12"}
```

Is this the incorrect way to query for geospatial data?

### PR DESCRIPTION
**Pull request by phated**
Still needs tests, but am I on the right track with this?

### CODE REVIEW

File: lib/superagent_transport.js [hunk 1]
```diff
@@ -42,6 +42,15 @@ SuperagentTransport.prototype.get_text = function get_text(path, callback) {
     });
 }
 
+SuperagentTransport.prototype.get_request = function put_request(url, json_obj, callback) {
```
Comment (dmitrizagidulin) [inline lib/superagent_transport.js]: Is the `prototype.get_request` = `put_request()` mismatch on purpose? And anyways, you can probably just use `transport.get_body()` instead, no?

**Comment by dmitrizagidulin** [comment]
Yeah, that's it! 
Just let me know what you meant to do re the get_request() above, and I'll merge it.

File: lib/superagent_transport.js (hunk 1 above)
Comment (phated) [inline lib/superagent_transport.js]: It was a copy-paste mistake, I will fix that.  I am not yet sure if it makes sense to only get the body back or the whole request, so I implemented it in this fashion.

**Comment by phated** [comment]
The `get_request` function was added because the other `get_*` methods don't take query parameters.  I fixed the typo of `put_request`.

I added an integration test for the `raw_solr_query` method and cleaned up some tests + the seed file to wait for the `set_schema` callback before proceeding.  Also, increased the timeout for a slow dev instance.

**Pull request merged**